src/store/reducers/todo.js

```diff
@@ -9,7 +9,29 @@ import { TASK_DATE_FORMAT } from "utils/const";
 import { findTaskIndexById, generateTaskObject } from "utils/helper";
 
 const initialTodoState = {
-  tasks: [generateTaskObject("Complete Initial setup of the Todo App")],
+  tasks: [
+    generateTaskObject("Complete Initial setup of the Todo App 1"),
+    generateTaskObject("Complete Initial setup of the Todo App 2"),
+    generateTaskObject("Complete Initial setup of the Todo App 3"),
+    generateTaskObject("Complete Initial setup of the Todo App 4"),
+    generateTaskObject("Complete Initial setup of the Todo App 5"),
+    generateTaskObject("Complete Initial setup of the Todo App 6"),
+    generateTaskObject("Complete Initial setup of the Todo App 7"),
+    generateTaskObject("Complete Initial setup of the Todo App 8"),
+    generateTaskObject("Complete Initial setup of the Todo App 9"),
+    generateTaskObject("Complete Initial setup of the Todo App 10"),
+    generateTaskObject("Complete Initial setup of the Todo App 11"),
+    generateTaskObject("Complete Initial setup of the Todo App 12"),
+    generateTaskObject("Complete Initial setup of the Todo App 13"),
+    generateTaskObject("Complete Initial setup of the Todo App 14"),
+    generateTaskObject("Complete Initial setup of the Todo App 15"),
+    generateTaskObject("Complete Initial setup of the Todo App 16"),
+    generateTaskObject("Complete Initial setup of the Todo App 17"),
+    generateTaskObject("Complete Initial setup of the Todo App 18"),
+    generateTaskObject("Complete Initial setup of the Todo App 19"),
+    generateTaskObject("Complete Initial setup of the Todo App 20"),
+    generateTaskObject("Complete Initial setup of the Todo App 21"),
+  ],
 };
```

Comment:
const numTasks = 21;
const tasks = Array.from({ length: numTasks }, (_, i) =>
  generateTaskObject(`Complete Initial setup of the Todo App ${i + 1}`)
);

const initialTodoState = {
  tasks,
};